openbas-front/src/admin/components/atomic_testings/atomic_testing/AtomicTestingUpdate.tsx

```diff
@@ -52,6 +53,7 @@ const AtomicTestingUpdate: FunctionComponent<Props> = ({
       ]),
     )(data);
     updateAtomicTesting(atomic.inject_id, toUpdate).then((result: { data: InjectResultDTO }) => {
+      MESSAGING$.notifySuccess('The element has been updated');
```

Comment:
Should be put in the simplePutCall direclty

Reply:
Ah yes we can have same as the errors a default alert for POST PUT & DELETErequests (not for get because 💥) with the same defaultBehavior system that is true by default and that can be override if needed ?